### PR TITLE
ACTIVE-814 add partners component

### PR DESCRIPTION
**Pull request by tomerye**
add partners component
image

### CODE REVIEW

File: src/components/FbxPartnersLogos/FbxPartnersLogos.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# FbxPartnersLogos
```
Comment (forforeach) [inline src/components/FbxPartnersLogos/FbxPartnersLogos.md]: Add a short description of this component's purpose.

File: test/unit/jest.conf.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ module.exports = {
   ],
   moduleNameMapper: {
     '^@/(.*)$': '<rootDir>/src/$1',
+    '^.+\\.svg$': '<rootDir>/src/__mocks__/fileMock.js'
```
Comment (ItamarGronich) [inline test/unit/jest.conf.js]: While not now, We would need to expand this in the future `(.svg|.png|.gif|.jpg)` and so on...

Comment (nlitwin) [reply test/unit/jest.conf.js]: If we wan't to copy what we're doing in frontend, we have:

```js
  transform: {
    "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
    ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest",
    "^.+\\.css$": "<rootDir>/tests/unit/cssTransform.js",
    "^(?!.*\\.(js|jsx|mjs|css|json)$)": "<rootDir>/tests/unit/fileTransform.js",
  },
```

and in `fileTransform.js`:

```js
const path = require("path");

// This is a custom Jest transformer turning file imports into filenames.
// http://facebook.github.io/jest/docs/en/webpack.html

module.exports = {
  process(__src, filename) {
    return `module.exports = ${JSON.stringify(path.basename(filename))};`;
  },
};

```

Basically any file that is not `js, vue, css, jsx, mjs, css, json` etc. uses the `fileTransform`

File: src/components/FbxPartnersLogos/logos/Synchrony.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="360px" height="99px" viewBox="0 0 360 99" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
```
Comment (nlitwin) [inline src/components/FbxPartnersLogos/logos/Synchrony.svg]: `svgo` can get rid of these extra lines added by Sketch

File: test/unit/jest.conf.js (hunk 2 above)
Comment (ItamarGronich) [reply test/unit/jest.conf.js]: Nice ^

**Pull request merged**